### DISCUSSION TITLE
直播助手里的更换勋章可以支持超过500个吗？

### DISCUSSION THREAD

**Discussion by Malus-Mill** [the1812/Bilibili-Evolved#3239] [功能建议 / Ideas]
如题，表示粉丝勋章超过500了、多余的就不显示出来
![image](https://user-images.githubusercontent.com/53999738/162714503-e017e9e5-bdbb-4442-84c1-4c2fabeb4e7e.png)
![image](https://user-images.githubusercontent.com/53999738/162714676-c7e64d07-a5da-48ad-bc47-8aec18395071.png)
下图的查水表就显示不出来
![image](https://user-images.githubusercontent.com/53999738/162714777-6657374e-d113-453a-a10a-6c0ab779b14b.png)
![image](https://user-images.githubusercontent.com/53999738/162714818-9dae5b5e-7fd0-4e97-8a90-7d0ed1abb1d5.png)

**Comment by the1812** [comment]
这么多显示出来也很难找吧...

**Reply by Malus-Mill** [reply]
熟练运用ctrl+F就可以，其实主要去的直播间比较少，表示列出来比一个一个翻列表好多了
